docs/conf.py

```diff
@@ -0,0 +1,28 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Zino'
+copyright = '2024, Morten Brekkevold, Johanna England, Simon Tveit'
```

Comment:
We cannot claim personal copyright here :-)

Either

```suggestion
copyright = '2024, Sikt'
```

or 


```suggestion
copyright = '2024, Sikt - The Norwegian Agency for Shared Services in Education and Research'
```